Makefile

```diff
@@ -131,16 +131,16 @@ crd-gen:
 
 .PHONY: ubuntu
 ubuntu:
-	@echo "===> Building antrea-ubuntu Docker image <==="
-	docker build -t antrea-ubuntu -f build/images/Dockerfile.ubuntu .
-	docker tag antrea-ubuntu antrea-ubuntu:$(DOCKER_IMG_VERSION)
+	@echo "===> Building antrea/antrea-ubuntu Docker image <==="
+	docker build -t antrea/antrea-ubuntu -f build/images/Dockerfile.ubuntu .
+	docker tag antrea/antrea-ubuntu antrea/antrea-ubuntu:$(DOCKER_IMG_VERSION)
```

Comment:
I should adjust the e2e test scripts to use antrea/antrea-ubuntu, then.

Reply:
Fixed by @edwardbadboy